pages/nft-book-store/user/index.vue

```diff
@@ -304,7 +304,17 @@ async function loadCommissionHistory () {
     if (fetchError.value && fetchError.value?.statusCode !== 404) {
       throw new Error(fetchError.value.toString())
     }
-    commissionHistory.value = (data.value as any)?.commissions || []
+
+    commissionHistory.value = (data.value as any)?.commissions?.map((item: any) => {
+      switch (item.type) {
```

Comment:
Please swap in computed

Reply:
updated
[445ae81](https://github.com/likecoin/nft-book-press/pull/149/commits/445ae8106f3846647ef29dc5719cb45823d7ff57)